R/mutate.R

```diff
@@ -175,17 +175,17 @@ mutate_as_date <- function(data, vars, names = NULL) {
 #' with tidyverse code attached
 #' @seealso \code{\link{code}}
 #' @examples
-#' created <- mutate_new(
+#' created <- new_vars(
 #'     data = iris,
 #'     vars = "Sepal.Length_less_Sepal.Width",
 #'     "Sepal.Length - Sepal.Width"
 #' )
 #' cat(code(created))
 #' head(created)
-#' @rdname mutate_new
+#' @rdname new_vars
 #' @author Stephen Su
 #' @export
-mutate_new <- function(data, vars = ".new_var", vars_expr = NULL) {
+new_vars <- function(data, vars = ".new_var", vars_expr = NULL) {
```

Comment:
I think I'd prefer to call this one `create_vars()`, [as per the GUI](https://inzight.nz/user_guides/variables/#create).